### ISSUE TITLE
Cannot open repository-folder in Jetbrains Phpstorm

### ISSUE THREAD

**Issue by antond1990** [gitpod-io/gitpod#10553]
### Bug description

It opens allways the folder of the main repository where the gitpod main configuration file is, not the folder of the project's repository I opened in Gitpod.

### Steps to reproduce

```
additionalRepositories:
  - url: https://github.com/xxx/repo1
    checkoutLocation: repo1
  - url: https://github.com/xxx/repo2
    checkoutLocation: repo2

tasks:
  - name: gitpod-setup
    init: |
      cd /workspace/gitpod
      bash setup-gitpod.sh

workspaceLocation: /workspace/gitpod/xxx.code-workspace

vscode:
  extensions:
    - recca0120.vscode-phpunit
    - ckolkman.vscode-postgres
    - msjsdiag.vscode-react-native
    - dbaeumer.vscode-eslint

github:
  prebuilds:
    # enable for the master/default branch (defaults to true)
    master: true
    # enable for all branches in this repo (defaults to false)
    branches: true
    # enable for pull requests coming from this repo (defaults to true)
    pullRequests: true
    # enable for pull requests coming from forks (defaults to false)
    pullRequestsFromForks: true
    # add a "Review in Gitpod" button as a comment to pull requests (defaults to true)
    addComment: true
    # add a "Review in Gitpod" button to pull requests (defaults to false)
    addBadge: false
    # add a label once the prebuild is ready to pull requests (defaults to false)
    addLabel: prebuilt-in-gitpod
```

### Expected behavior

Jetbrains Phpstorm opens the folder of the correct repository.

### Anything else?

<img width="455" alt="grafik" src="https://user-images.githubusercontent.com/24891375/172831226-206c1bbb-a743-4d8d-90e5-39b3e71384e5.png">

The workspace of repo1 was opened, but Phpstorm only shows the gitpod repo.

```
JetBrains Gateway 2022.1.3
Build #GW-221.5787.23, built on May 30, 2022
Runtime version: 11.0.15+10-b2043.56 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 12.2.1
GC: G1 Young Generation, G1 Old Generation
Memory: 512M
Cores: 12
Non-Bundled Plugins:
    io.gitpod.jetbrains.gateway (0.0.1-main.3498)
```

**Comment by loujaybee** [comment]
Thanks @antond1990, is this a duplicate of: 

- https://github.com/gitpod-io/gitpod/issues/6900 ?
